applications/consdb/values-summit.yaml

```diff
@@ -0,0 +1,11 @@
+config:
+  site_id: summit
+  nfs_path_1: /repo/LSSTComCam  # Mounted as /volume_1
+  nfs_server_1: comcam-archiver.cp.lsst.org
+  butler_uri_1: /volume_1
+
+  nfs_path_2: /auxtel/repo/LATISS  # Mounted as /volume_2
+  nfs_server_2: nfs-auxtel.cp.lsst.org
+  butler_uri_2: /volume_2
```

Comment:
Are you using this `config` map? I don't see this in `values.yaml` and I don't see where these are used. If its stale, you might want to delete it.

Reply:
KT is not using config map inside consdb code - i can work on that with him  these are indeed not needed i removed them